src/routing/routes.tsx

```diff
@@ -0,0 +1,13 @@
+import { Routes, Route } from "react-router-dom";
+
+import Home from "src/views/home/home.view";
+
+function Router(): JSX.Element {
```

Comment:
The file is named `routes.tsx` but the component is called `Router`, which is the correct one? `Router` looks better.